docker-compose.yml

```diff
@@ -10,11 +10,11 @@ services:
     #network_mode: "host"
     environment:
       NODE_ENV: "production" 
-      PROXY_DB: "mongodb://web-proxy-mongo/proxy"
-      PROXY_KEY: "a"
-      PROXY_SALT: "a"
-      #PROXY_OVH_KEY: ""
-      #PROXY_TRACE_KEY: ""
+      PROXY_MONGODB_ADDON_URI: "mongodb://web-proxy-mongo:27017/proxy"
+      PROXY_JWT_SECRET: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+      PROXY_PEPPER: "a"
+      PROXY_JWT_ISSUER: "aaa"
+      PROXY_JWT_AUDIENCE: "bbbb"
```

Comment:
App won't start if it's empty, but I agree with you, we can put better temporary passwords